docs/client/request.md

```diff
@@ -211,6 +211,37 @@ Header method returns header value via key, this method will visit all field in
 func (r *Request) Header(key string) []string
 ```
 
+### Headers
```

Comment:
General comment for docs:

We mention in multiple places the usage of `maps.Collect()` but never provide an example. Probably worth adding an example to avoid follow-on questions from users.